rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -2527,6 +2590,7 @@ void PPUTranslator::LDX(ppu_opcode_t op)
 
 void PPUTranslator::LWZX(ppu_opcode_t op)
 {
+	m_may_be_mmio &= (op.ra != 1u && op.ra != 13u && op.rb != 1u && op.rb != 13u);
```

Comment:
What does register check signify here and in similar places?

Reply:
Stack register and TLS address register are unlikely to be used in MMIO address calculation.

Reply:
Added it as comment.